auth/src/Controller/authController.js

```diff
@@ -66,20 +66,12 @@ class AuthController {
   async setcookie (req, res) {
     try {
       const userToken = req.headers.authorization
-      res.setHeader('Set-Cookie', `token=${userToken}; Path=/; HttpOnly; Max-Age=3600`)
+      res.setHeader('Set-Cookie', `token=${userToken}; Path=/; HttpOnly; Max-Age=${tenYearsInSeconds}`)
```

Comment:
- Consider the SameSite Attribute: The SameSite attribute controls whether a cookie is sent with cross-origin requests, providing some protection against cross-site request forgery (CSRF) attacks. You can set it to Strict, Lax, or None. Strict is the most secure option, but it might be too restrictive for cookies that need to be sent in a cross-site context. Lax is a good balance for most cases
- Use Secure Flag: If your site is served over HTTPS (which it should be for security reasons), add the Secure flag to ensure the cookie is sent only over secure channels. This helps prevent the cookie from being intercepted by attackers
- Regarding `tenYearsInSeconds` try enhancing user experience without compromising security, consider implementing sliding sessions. This involves renewing the session token (and its Max-Age) each time the user interacts with the server. If the user remains active, their session stays alive, but once they're inactive for a specified duration, the session expires. This keeps sessions short for inactive users while avoiding frequent re-authentication for active users